src/applications/vass/sass/vass.scss

```
@@ -1,7 +1,20 @@
+@import "~@department-of-veterans-affairs/css-library/dist/tokens/scss/variables";
+
 .vass-form__button-container {
   row-gap: 16px;
+  @media (max-width: $small-screen) {
+    column-gap: 8px;
```

Do you think we could use the spacing tokens ?
```suggestion
    column-gap: var(--vads-spacing-1);
```